hashin.py

```diff
@@ -30,6 +31,33 @@
             "Consider upgrading your version of Python."
         )
 
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    'packages',
+    help="One or more package specifiers (e.g. some-package or some-package==1.2.3)",
+    nargs='+'
+)
+parser.add_argument(
+    '-r', '--requirements-file',
+    help="requirements file to write to (default requirements.txt)",
+    default='requirements.txt'
+)
+parser.add_argument(
+    '-a', '--algorithm',
+    help="The hash algorithm to use: one of sha256, sha384, sha512",
+    default='sha256'
+)
+parser.add_argument(
+    '-v', '--verbose',
+    help="Verbose output",
+    action="store_true",
+)
+parser.add_argument(
+    '-p', '--python-version',
+    help='Python version to add wheels for. May be used multiple times.',
+    action='append',
+    default=[],
+)
```

Comment:
that's generally when you want to run these, it just creates an object it doesn't parse any arguments.